File: app/src/main/java/com/minirogue/starwarscanontracker/model/room/pojo/MediaAndNotes.java
package com.minirogue.starwarscanontracker.model.room.pojo;

import androidx.room.Embedded;

import com.minirogue.starwarscanontracker.model.room.entity.MediaItem;
import com.minirogue.starwarscanontracker.model.room.entity.MediaNotes;

public class MediaAndNotes {

    @Embedded
    public MediaItem mediaItem;
    @Embedded
    public MediaNotes mediaNotes;
}

File: core/src/main/java/com/minirogue/starwarscanontracker/core/model/room/pojo/MediaAndNotes.kt
package com.minirogue.starwarscanontracker.core.model.room.pojo

import androidx.room.Embedded
import com.minirogue.starwarscanontracker.core.model.room.entity.MediaItemDto
import com.minirogue.starwarscanontracker.core.model.room.entity.MediaNotesDto

data class MediaAndNotes(@Embedded val mediaItemDto: MediaItemDto, @Embedded val mediaNotesDto: MediaNotesDto)
